### PR TITLE
Redesign app

### PR DESCRIPTION
**Pull request by pokiujf**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,16 @@
...
+    "@fortawesome/fontawesome-svg-core": "^1.2.8",
+    "@fortawesome/free-solid-svg-icons": "^5.5.0",
+    "@fortawesome/react-fontawesome": "^0.1.3",
     "bcrypt": "^3.0.2",
     "fetch-mock": "^7.2.0",
+    "i": "^0.3.6",
     "json-server": "^0.14.0",
     "jsonwebtoken": "^8.3.0",
     "moment": "^2.22.2",
+    "npm": "^6.4.1",
```
Comment (pokiujf) [inline package.json]: why do we need npm dependency?

File: src/components/Dashboard/Inventory/AdminProduct.js [hunk 2]
```diff
@@ -36,11 +36,11 @@ export class AdminProduct extends Component {
     if (this.state.toggleEdit) {
       return (
         <React.Fragment>
```
Comment (pokiujf) [inline src/components/Dashboard/Inventory/AdminProduct.js]: No need to have Fragment here, as we are returning one element

File: src/components/Dashboard/Shop/Product.js [hunk 3]
```diff
@@ -92,46 +101,51 @@ export class Product extends Component {
...
+            </span>
           </p>
           <p id="stock-error">{error}</p>
-          <p className="product-item__total">
-            <span>Total price:</span>
+          <p className="product-desc__total">
+            Total price:
             <span id="total-price">
-              <span>$</span>
+$
```
Comment (pokiujf) [inline src/components/Dashboard/Shop/Product.js]: fix indentation

File: src/styles/sidebar.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+.sidebar__list .item a {
+    text-decoration: none;
+    color: rgb(167, 162, 162);
+    font-size: 24px;
+    
+}
+
+.sidebar__list .item a:hover {
+    color: rgb(219, 212, 212);
+
```
Comment (pokiujf) [inline src/styles/sidebar.css]: There is a lot of codestyle issues in css.
Empty lines, inconsistent spaces between declarations.

It gives bad first impression.
Please fix that, and add tralling empty line in the end of the files where it's missing

Comment (hrodberaht) [reply src/styles/sidebar.css]: I rewrote everything to sass, and improved in next branches.

Comment (hrodberaht) [reply src/styles/sidebar.css]: I rewrote everything to sass, and improved in next branches.

**Pull request merged**